merge-all.rb

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env ruby
+
+# Usage:
+# "REPOS_PATH=infrastructure GH_ACCESS_TOKEN=abc123 ./merge-all.rb"
```

Comment:
it might be useful to indicate in comments that all PRs have to be green before it does any merging -- I thought it merged all but the failing PRs.  :-P